scripts/ci/render_bench.py

```diff
@@ -282,6 +282,7 @@ def includes(self, other: Target) -> bool:
 
     def render(self, gcs: storage.Client, after: datetime) -> dict[str, str]:
         commits = get_commits(after)
+        print("commits", commits)
```

Comment:
This is exactly what I meant by printing often, and I didn't follow my own advice 😄.